final/evaluation.tex

```diff
@@ -180,14 +180,22 @@ \subsection{Development Methodologies}
 Naturally, there were also things that could have gone better. It is easy to
 underestimate the amount of work when putting together a sprint plan, which has
 been a source of problems. The solution for this issue is to deliberately plan
-less: it is easier to move new user stories to the sprint plan than it is to decide
-which user stories not to complete.
-
-Another issue is that the development was struck with two sources of delay.
-Spoofax changes fast, thus several times development had to stop to catch up
-with these changes. The second source of delay was the integration with the
-Eclipse IDE, which led to several unforeseen issues. Some time could have been
-saved if the client had been asked for advice sooner.\\
+less: it is easier to move new user stories to the sprint plan than it is to
+decide which user stories not to complete.
+
+A second issue concerns the unstable development environment provided by Eclipse.
+Omitting all the details, several hard to track issues occurred which took a
```

Comment:
`Waste` is a negative word. What I'm saying is you should just write:

We experienced a lot of delay due to the introduction of a new Dynsem implementation, since this renewed implementation could not be used in our design as is. We therefore spent considerable time trying to refactor parts of the new codebase for better interopability in the context of a REPL. These refactored parts have been contributed back to the Dynsem upstream repository, where they've been accepted and published in the latest snapshot releases.

No negative charge at all, but still explaining we had a large delay.